scripts/proposals/AIP12/data/42161-AIP1.2-data.json

```diff
@@ -0,0 +1,9 @@
itutional - Process\n\nAbstract:\nThis document (“AIP-1.2”) proposes amendments to the Constitution, and The Arbitrum Foundation Amended & Restated Memorandum & Articles of Association (the “A&R M&A”) and Bylaws (the “Bylaws”) to (1) remove references to AIP-1, and (2) make other changes reflecting feedback from the community.\n\nMotivation:\nAIP-1 set out critical aspects of governance and included key governance documents for the ArbitrumDAO, and The Arbitrum Foundation which referenced AIP-1 throughout: the ArbitrumDAO Constitution (the “Constitution”), the Bylaws and the A&R M&A. However, after vigorous community debate, AIP-1 did not pass.\n\nRationale:\nThe Constitution is a foundational document that lays out the governance system and capabilities of the ArbitrumDAO; the Bylaws and A&R M&A outline The Arbitrum Foundation’s relationship with and obligations to the ArbitrumDAO. These documents should be updated, via amendment, to remove references to AIP-1 and also to reflect other changes requested by the community during the debate process over AIP-1.\n\nSpecifications:\n\nAmendments to the Constitution (https://drive.google.com/file/d/1pIYxg9rJzIPcP0bvQAaPgmLITM9rVBIf/view?usp=share_link)\n-Remove references to AIP-1, except as reference to the moment the original Constitution became effective.\n-Make explicit that proposals to amend The Arbitrum Foundation’s A&R M&A and Bylaws take the form of a Constitutional AIP.\n-Make explicit that the DAO may make a Non-Constitutional Funding AIP with respect to the Administrative Budget Wallet.\n-Lower the threshold number of Votable Tokens required for an AIP to be posted on-chain from 5,000,000 $ARB to 1,000,000 $ARB.\n-Add a new Section 5, which details the Data Availability Committee, including processes for removing and appointing Data Availability Committee members.\n\nAmendments to the Bylaws (https://drive.google.com/file/d/1VLYNk9VYUWMl0sL2wchot6I6arHCzd4r/view?usp=share_link)\n-Remove references to AIP-1.\n-Clarify that the definition of “Administrative Budget Wallet” includes all assets that are contributed to or otherwise acquired by the Administrative Budget Wallet, inclusive of assets specifically approved by the ArbitrumDAO. As pointed out by the DAO in its feedback in response to AIP-1, the “Administrative Budget Wallet” defined term in the Bylaws was unclear and didn’t clearly reflect the reality of The Arbitrum Foundation’s receipt of 7.5% of the token supply upon the $ARB token genesis.\n-Remove references to “Special Grants” and replace with the concept of Arbitrum ecosystem growth.\n-Remove “AIP Threshold” defined term as it is not used elsewhere in the Bylaws\n-Clarify that the ArbitrumDAO may replace The Arbitrum Foundation’s directors, change the number of directors, and require The Arbitrum Foundation to take certain actions.\n\nAmendments to the A&R M&A (https://drive.google.com/file/d/1Zmi5w21skdwuC7EAe59mYBC_4fS1vqzJ/view?usp=share_link)\n-Remove references to AIP-1.\n",
```

Comment:
It's probably ok as is, given that the changes have been made to the actual consitution doc, but should we have more permanent references than google drive?

Reply:
i think we want to keep the proposal text in line with what was in the snapshot (the more permanent location is on gh)